tests/basic/03_dependencies.py

```diff
@@ -6,7 +6,8 @@
     title="pattern-with-dependencies",
     entrypoint="pattern_with_dependencies.py",
     working_dir="./source_files/",
-    dependencies=["qiskit-experiments==0.6.0"],
```

Comment:
It seems `qiskit-experiments` is not working with the newer Qiskit version so I changed it by now for another thing to test dependencies installation. I accept other proposals.

Reply:
I believe that the `docs/getting_started/basic/03_dependencies.ipynb` also has to be updated.

Reply:
It's better to revisit this test/doc.  Would you open an issue for that?

Reply:
Fixed here: [92cf627](https://github.com/Qiskit/qiskit-serverless/pull/1331/commits/92cf627a5a931cbe6b541f35354d1a8ec60ff617)

Yep, I will open an issue now to review it 👍

Reply:
Issue opened #1339